genai_stack/stack/mediator.py

```diff
@@ -40,6 +40,9 @@ def _is_component_available(self, component_name:str) -> bool:
     def get_embedded_text(self, text):
         return self._stack.embedding.embed_text(text)
 
+    def get_embedding_function(self):
+        return self._stack.embedding.embedding
```

Comment:
Why its embedding.embedding are you sending a embedding variable to mediator maybe you can directly return it so that you can access directly or you can change variable name

Reply:
No embedding.embedding ==> embedding[Component].embedding[actual langchain client in the component]